turborepo-vercel/apps/remix-app/package.json

```diff
@@ -5,7 +5,8 @@
   "sideEffects": false,
   "scripts": {
     "build": "remix build",
-    "dev": "remix dev"
+    "dev": "remix dev",
+    "typecheck": "tsc -b"
```

Comment:
looks like these example is using `isbot` but doesn't have the dep, however vercel doesn't support streaming, can you change the `entry.server` to the renderToString version 🙏

Reply:
PR created: https://github.com/remix-run/examples/pull/99